### ISSUE TITLE
Load models from NLP cluster

### ISSUE THREAD

**Issue by nickatomlin** [Designist/Negotiation#16]
Cannot seem to load the models from the NLP cluster:

```
InvalidArgumentError: Input 1 of node decoder/while/Merge_1_1 was passed int32 from decoder/while/NextIteration_1:0 incompatible with expected float.
```

**Comment by nickatomlin** [comment]
(In general, loaded models seem to perform worse.)

**Comment by nickatomlin** [comment]
Fixed in ffe48cc5c61171940e3ae74d804cc5a722c42076
